_data/simple-icons.json

```
@@ -7074,6 +7074,12 @@
             "hex": "FF6801",
             "source": "https://www.joinhoney.com"
         },
+        {
+            "title": "Honeybadger",
+            "hex": "EA5A37",
```

```suggestion
            "hex": "EA5937",
```

Suggestion is the hex code given on the Assets page.